erdgo/errors.go

```diff
@@ -0,0 +1,9 @@
+package erdgo
+
+import "errors"
+
+var (
+	errInvalidPubkey  error = errors.New("Invalid pubkey")
```

Comment:
Usually, we start with lowercase for error messages.

Reply:
Done